src/apps/integrations/service/integrations.py

```diff
@@ -0,0 +1,31 @@
+import json
+
+from apps.integrations.domain import AvailableIntegrations, Integration
+from apps.integrations.errors import UniqueIntegrationError
+from apps.users.domain import User
+from apps.workspaces.crud.workspaces import UserWorkspaceCRUD
+
+__all__ = [
+    "IntegrationService",
+]
+
+
+class IntegrationService:
+    def __init__(self, session, user: User) -> None:
+        self.session = session
+        self.user = user
+
+    async def enable_integration(self, integrations: list[Integration]):
+        workspace = await UserWorkspaceCRUD(self.session).get_by_user_id(user_id_=self.user.id)
```

Comment:
Is it requirement that workspace owner the only person who can manage integrations?

Reply:
For now, there is no exact requirements regarding workspace integration accesses